portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -32,6 +34,14 @@ object CromwellPipelineApp extends App {
     routeCombiner(userController.route, projectController.route, projectFileController.route)
   }
 
+  implicit def rejectionHandler =
+    RejectionHandler.default.mapRejectionResponse {
+      case res @ HttpResponse(_, _, ent: HttpEntity.Strict, _) =>
+        val message = ent.data.utf8String.replaceAll("\"", """\"""")
+        res.withEntity(HttpEntity(ContentTypes.`application/json`, message))
+      case x => x
+    }
+
```

Comment:
This rejection handler basically does nothing. This is not what we discussed.

Reply:
Done